lib/ui/archive_page.dart

```diff
@@ -8,15 +8,18 @@ import 'package:photos/models/magic_metadata.dart';
 import 'package:photos/models/selected_files.dart';
 import 'package:photos/ui/gallery.dart';
 import 'package:photos/ui/gallery_app_bar_widget.dart';
+import 'package:photos/ui/gallery_overlay_widget.dart';
 
 class ArchivePage extends StatelessWidget {
   final String tagPrefix;
   final GalleryAppBarType appBarType;
+  final GalleryOverlayType overlayType;
```

Comment:
Do we need another property called `GalleryOverlayType` or we can just use `GalleryAppBarType`. We can may be rename it to `GalleryType`?

Reply:
Yes we can rename it to GalleryType